src/clients/base-evernode-client.js

```diff
@@ -327,11 +327,11 @@ class BaseEvernodeClient {
 
         return null;
     }
-}
 
-function getStateData(states, key) {
-    const state = states.find(s => key === s.key);
-    return state?.data;
+    getStateData(states, key) {
```

Comment:
This previously existed as an external function to avoid being tied to the Client instance as well as avoid exposing it via the class's public interface.